honeybee_vtk/helper.py

```diff
@@ -0,0 +1,147 @@
+"""Functions to help in geometry translation."""
+
+from ladybug_geometry.geometry3d import Face3D, Point3D
+
+
+def get_mesh_points(boundary_points, holes_points=None):
+    """Convert a Face3D with holes in triangulated mesh and get vertices.
+
+    This function constructs a Ladybug Face3D object from vertices and creates
+    triangulated meshes from the generated Face3D object. The function then returns
+    the vertices of these generated meshes.
+
+    Args:
+        boundary_points: A list of points that represent the boundary of a face. Here,
+            each point is a list of X, Y, and Z cordinates of the point.
+        holes_points: A list of lists of points that represent the boundary of the holes
+            in the face. Here, each point is a list of X, Y, and Z cordinates of the
+            point. Defaults to None. Which will mean the face has no holes.
+
+    Returns:
+        A list of list of points. Each list has vertices for a triangular polygon.
+        Example of structure will be [[X, Y, Z], [X, Y, Z], [X, Y, Z]].
+    """
+    # Create Ladybug Point3D objects from boundary points
+    lb_boundary_pts = [
+        Point3D(point[0], point[1], point[2])
+        for point in boundary_points]
+
+    # Create a Ladybug Face3D object from boundary points and hole points
+    if holes_points:
+        # Create Ladybug Point3D objects from hole points
+        lb_holes_pts = [
+            [Point3D(point[0], point[1], point[2]) for point in points]
+            for points in holes_points]
+
+        face = Face3D(boundary=lb_boundary_pts, holes=lb_holes_pts)
+
+    else:
+        face = Face3D(boundary=lb_boundary_pts)
+
+    # Generate triangles from the Face3D object
+    triangles = face.triangulated_mesh3d
+
+    # A list of lists with each list having vertices of triangles
+    points_lst = [[
+        triangles.vertices[face[0]], triangles.vertices[face[1]],
+        triangles.vertices[face[2]]] for face in triangles.faces]
+
+    return points_lst
+
+
+def check_convex(boundary_points):
+    """Check whether a Face3D is convex or not.
+
+    This function creates a Ladybug Face3D object from boundary points and reports
+    whether the Face3D is convex or not.
+
+    Args:
+        boundary_points: A list of list of points. Here, each point is a list of 
+            X, Y, and Z cordinates of the point.
+
+    Returns:
+        A boolean value.
+    """
+
+    lb_boundary_pts = [
+        Point3D(point[0], point[1], point[2]) for point in boundary_points]
```

Comment:
I'm not going to comment on all the instances of code like this. Refer to the one earlier.